app/src/test/java/com/losandroides/learn/domain/ItemsUseCaseTest.kt

```diff
@@ -0,0 +1,34 @@
+package com.losandroides.learn.domain
+
+import com.losandroides.learn.framework.coVerifyOnce
+import com.losandroides.learn.framework.relaxedMockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+class ItemsUseCaseTest {
+
+    private val testDispatcher = StandardTestDispatcher()
+    private val itemRepository: ItemRepository = relaxedMockk()
+
+    private lateinit var useCase: ItemsUseCase
+
+    @Before
```

Comment:
Thanks for the insight, @joancolmenerodev.
I am checking it is only called once.
Also, why check this on the `tearDown()`?

Reply:
It's like saying, once every test has been tested no more interactions with the mocks. (to avoid missing some calls)
But perhaps this is for the future, since now as you said there's only one test. 😄